docs/ref/middleware.txt

```diff
@@ -411,6 +411,13 @@ Middleware ordering
 
 Here are some hints about the ordering of various Django middleware classes:
 
+#. :class:`~django.middleware.security.SecurityMiddleware`
+
+   For the moment, it only processes the head of request/response,
+   so it does not matter whether it is put after ``GZipMiddleware`` or before
```

Comment:
Not sure the reference to GZipMiddleware is really needed -- what do you think Carl?
Also, need to update the list in docs/topics/http/middleware.txt if we change the project template.